cmds/core/netcat/connect.go

```diff
@@ -60,9 +61,11 @@ func (c *cmd) connectMode(output io.Writer, network, address string) error {
 
 		// prepare command execution on the server
 		if c.config.CommandExec.Type != netcat.EXEC_TYPE_NONE {
-			if err := c.config.CommandExec.Execute(conn, io.MultiWriter(conn, output), c.stderr, c.config.Misc.EOL); err != nil {
```

Comment:
You remove the first argument in the signature of `CommandExec.Execute` which is stdin but here you drop the second parameter (stdout) of the invocation. Is this intended?

Reply:
Yes, the stdin was not required. But the multiwriter was removed, so the output is only written to the connection and not output (like ncat).